extension-androidauto/src/main/java/com/mapbox/maps/extension/androidauto/MapboxCarMapGestureHandler.kt

```diff
@@ -10,7 +10,7 @@ import com.mapbox.maps.ScreenCoordinate
  * with [MapboxCarMap.setGestureHandler].
  */
 @MapboxExperimental
-interface MapboxCarMapGestureHandler {
+abstract class MapboxCarMapGestureHandler {
```

Comment:
Am looking closer at `@JvmDefault`

Because that's exactly what we want for this case

Reply:
I changed this to a java interface instead..

Am also testing this compiler option
```
  kotlinOptions {
    freeCompilerArgs.plus("-Xjvm-default=all")
  }
```

Reply:
I can't get the `jvm-default` compiler flag to work.. It seems like it's still an option issue https://youtrack.jetbrains.com/issue/KT-4779/Generate-default-methods-for-implementations-in-interfaces

Converting this to a java interface seems reasonable

Reply:
https://github.com/gradle/gradle/issues/19750 Came across this when searching more about this `jvm-default` flag

Reply:
Another link to the definitions of these compiler flags
https://github.com/JetBrains/kotlin/blob/master/compiler/testData/cli/jvm/extraHelp.out#L60-L85